src/Examine.Lucene/Suggest/Directories/FileSystemSuggesterDirectoryFactory.cs

```diff
@@ -0,0 +1,34 @@
+using System.IO;
+using Examine.Lucene.Directories;
+using Lucene.Net.Index;
+using Lucene.Net.Store;
+using Directory = Lucene.Net.Store.Directory;
+
+namespace Examine.Lucene.Suggest.Directories
+{
+    public class FileSystemSuggesterDirectoryFactory : SuggesterDirectoryFactoryBase
+    {
+        private readonly DirectoryInfo _baseDir;
+
+        public FileSystemSuggesterDirectoryFactory(DirectoryInfo baseDir, ILockFactory lockFactory)
+        {
+            _baseDir = baseDir;
+            LockFactory = lockFactory;
+        }
+
+        public ILockFactory LockFactory { get; }
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done